package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zenko/cloudserver",
-  "version": "8.7.18",
+  "version": "8.7.11",
```

Comment:
why the downgrade in package.json ?
8.7.18 is already released (even 8.7.19, even though it was not bump'ed)